test/box-luatest/schema_sys_space_test.lua

```diff
@@ -0,0 +1,26 @@
+local server = require('luatest.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_all = function(lg)
+    lg.server = server:new({alias = 'master'})
+    lg.server:start()
+end
+
+g.after_all = function(lg)
+    lg.server:drop()
+end
+
+g.test_cluster_uuid_update_ban = function(lg)
+    lg.server:exec(function()
+        local uuid = require('uuid')
+        local _schema = box.space._schema
+        local msg = "Can't reset replica set UUID"
+        t.assert_error_msg_contains(msg, _schema.replace, _schema,
+                                    {'cluster', uuid.str()})
+        t.assert_error_msg_contains(msg, _schema.replace, _schema,
+                                    {'cluster', tostring(uuid.NULL)})
+        -- Fine to replace with the same value.
+        _schema:replace{'cluster', box.info.cluster.uuid}
```

Comment:
> Why do you allow replacing the same value? Is this used anywhere?

I didn't want to ban it on purpose. Because it wasn't banned before and keeping it working is very easy. I only banned the update as it is obviously illegal.

> For example, repeaded logging of 'cluster uuid ...' looks strange:

This is fixed in a next commit - `schema: make _schema space transactional`.

Reply:
Okay.